docs/topics/about-vscode-extension.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+
+[id='about-vscode-extension_{context}']
+= About the {ProductName} extension for {VscodePluginName}
+
+The {ProductName} ({ProductShortName}) extension provides assistance directly in {VscodePluginName} for developers making changes for a migration or modernization effort.
+
+The {ProductShortName} extension analyzes your projects using customizable rulesets, marks migration issues in the source code, provides guidance to fix the issues, and offers automatic code replacement, or Quick Fixes, if possible.
```

Comment:
I would not mention Quick Fixes here because most people will not read this section or remember it. Define Quick Fixes later in the document.